scripts/sync_branding.sh

```diff
@@ -21,7 +21,7 @@ echo $mobile_root
 cd "$mobile_root"
 
 # Please update the sha when valora branding updates are needed
-valora_branding_sha=d4f5c6cea5c6dcba2a18ba2c20487cce5e13441f
+valora_branding_sha=027dc2843a50e7f0b65d861dd3fd7172de28db3f
```

Comment:
i think this PR will be merged after https://github.com/valora-inc/wallet/pull/3554, which contains a more recent branding hash so this change shouldn't be needed